src/assets/station-templates/schneider-imredd.station-template.json

```diff
@@ -45,6 +45,11 @@
         "key": "WebSocketPingInterval",
         "readonly": false,
         "value": "60"
+      },
+      {
+        "key": "ReserveConnectorZeroSupported",
+        "readonly": false,
+        "value": "true"
```

Comment:
true or false ? Please set consistent  default values for all templates.

Reply:
I've used different values for simulating stations, which are capable of reserving connector 0 and other ones, which are not capable. What would you prefer? Then I'll change it consistently.

Reply:
According to the spec, it is an optional configuration key. It has not to set

Reply:
Default to `false` then seems to be the sensible default.